Sources/FeatherModuleKit/Extensions/QueryBuilder+Exists.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//
+//
+//  Created by Tibor Bodecs on 02/03/2024.
+//
+
+import FeatherDatabase
+
+extension DatabaseQueryGet where Row: KeyedDatabaseModel {
```

Comment:
IMHO: this should be a DatabaseQueryExists protocol and the function should return a Bool.